src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -101,7 +127,23 @@ export const _Chatbox: React.FC<ChatboxProps> = ({
 
   return (
     <div className="Chatbox">
-      <div className="Chatbox__messages">{renderedMessages}</div>
+      <div
+        className="Chatbox__messages-container"
+        ref={scrollableComponentRef}
+        id={"Chatbox_scrollable_div"}
+      >
+        <InfiniteScroll
+          dataLength={messagesRenderCount}
+          className={"Chatbox__messages-infinite-scroll"}
+          next={renderMore}
+          inverse={true}
```

Comment:
```suggestion
          inverse
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/e42865859e420ecae364e18c34b6e57fd52ed180

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1922/commits/e42865859e420ecae364e18c34b6e57fd52ed180